### DISCUSSION TITLE
Possible to use for speech to phonomemes?

### DISCUSSION THREAD

**Discussion by fire** [coqui-ai/STT#1838] [Q&A] [answered]
Is it possible to take speech and convert it to https://www.researchgate.net/figure/The-standard-set-of-visemes-specified-in-MPEG-4-and-related-phonemes_tbl1_4151888 instead of text?

**Comment by cdleong** [comment, marked as answer]
@fire going from speech to phoneme symbols is possible if the model is trained on that specifically I believe. But you may wish to check out models designed for this task, like https://github.com/xinjli/allosaurus/

**Reply by fire** [reply]
I was hoping to embed it into a game, so was looking for MIT compatible licenses -- like MPL.
